deploy/dashboards/ngsa.json

```diff
@@ -72,7 +72,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(NgsaAppDuration_bucket[1m])) by (category)",
+          "expr": "sum(rate(NgsaAppDuration_bucket[1m])) by (mode)",
```

Comment:
@jofultz - this will not update the view - you'll need to save grafana.db with the change and check-in